src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -67,7 +66,7 @@ When you run LDR in `immediate` mode, you cannot replicate a table with [foreign
     cockroach sql --url "postgresql://root@{node IP or hostname}:26257?sslmode=verify-full" --certs-dir=certs
     ~~~
 
-1. On **both** clusters, enable the `kv.rangefeed.enabled` cluster setting:
+1. If you are setting up bidirectional LDR, on **both** clusters, enable the `kv.rangefeed.enabled` cluster setting. If you are setting up unidirectional LDR, enable this setting on the **source** cluster:
```

Comment:
given that subsequent steps are written for a unidirectional setup, and that we ask the user in a bidi setup to rinse and repeat below, i think it might be clearer to just say the source needs to enable rangefeeds here.